Adds function to export/rebuild XML files, which are also added to src c...

NealHumphrey opened the pull request:
...ontrol. Also makes automatic import/export of VBA code an optional setting.

New menu commands to export XML from any open file, and rebuild XML from a selected folder. XML data contains all spreadsheet objects, which are currently not accessible in the VBA-only exporter. Function saves the xlsx or xlsm as .zip and unzips to a new folder.

Sorry, not up to date with current master - if you're interested in merging in I can rebase off the current master.

hilkoc commented:
Hi Neal,
Thanks for your pull request. I've had a quick look and the xml export functionality works great. Could you please rebase your commit onto the latest master?

btw: It sometimes may take a while before I reply. That's only because I don't have much time at the moment. Sorry for this.

Thanks Hilko

NealHumphrey commented:
Hey Hilko, 

Took me a while to get back to this. I've rebased onto your current master and squashed down to one feature commit. Feel free to merge in if you'd like!

Note that I also added a AUTO_SAVE constant to turn off auto-exporting (looks like another person who forked the repository did the same). In the current code it's set to keep the auto-export off, but if you want it back on for master you'll want to change that.

Neal

hilkoc commented:
Hi Neal,

I've merged in your pull request. Thanks for your contribution.

The auto_save constant had to be removed. It adds If conditions that are unnecessary.To disable the eventhandling, simply don't instantiate the eventlistener. See the code in the 'ThisWorkbook' class. It is disabled by default because excel cannot import the class modules correctly. (See issue #1 )
